proxy/src/main/java/ru/leymooo/botfilter/BotFilterCommand.java

```diff
@@ -52,6 +55,16 @@ public void execute(CommandSender sender, String[] args)
         {
             export( sender, args );
             sender.sendMessage( "§aКоманда выполнена" );
+        } else if ( args[0].equalsIgnoreCase( "generate" ) )
```

Comment:
For what is the command? The captchas are being generated from the startup anyway

Reply:
> For what is the command? The captchas are being generated from the startup anyway

Custom ability to regenerate captcha at any time. There is an option for automatic regeneration, so I thought it would be nice to manually regenerate when needed.

Reply:
I don't know if you should really add this. Some people might don't get it and just spam the command because nothing happens. You could at least provide some type of boolean so that you can't spam it until it is done.

Reply:
> I don't know if you should really add this. Some people might don't get it and just spam the command because nothing happens. You could at least provide some type of boolean so that you can't spam it until it is done.

There is an exception interception, and an error message output to the user if the captcha is already in the process of being generated. The command is only for the console, players cannot execute it

Reply:
Oh ok, sorry for that haha